utils/kafka.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/segmentio/kafka-go"
+)
+
+func PublishMessage(kafkaURI, topic string, message string) bool {
```

Comment:
instead of returning a bool can you please return an error? this will make more sense.